examples/chat/src/libp2p-bundle.js

```diff
@@ -18,7 +18,7 @@ class Node extends libp2p {
       modules: {
         transport: [Memory],
         streamMuxer: [ mplex ],
-        connEncryption: [ NOISE, SECIO ]
+        connEncryption: [ SECIO ]
```

Comment:
For debugging is easier to used SECIO for now. It is going to be deprecated, once we have this working we should change to just NOISE